src/core/components/header/index.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+import { Header } from '../../../common/ui-kit'
+
+function PageHeader() {
+  return <Header style={{ background: '#fff', padding: 0 }} />
```

Comment:
Maybe it would be better if you added a file for example Header.scss for each component imported from antd in the future your styles may be more

Reply:
Read my comment above